amethyst_assets/src/asset.rs

```diff
@@ -104,6 +104,16 @@ pub trait Format<D: 'static>: objekt::Clone + Debug + Send + Sync + 'static {
 
 objekt::clone_trait_object!(<D> Format<D>);
 
+/// SerializableFormat is a marker trait which is required for Format types that are supposed
+/// to be serialized.
```

Comment:
Please document that this trait indicates both `Deserialize` and `Serialize` support, and that it should be implemented via `register_format!`.
If you want to add one, a doc example would be great here, too.